packages/pulp/pulp.spec

```diff
@@ -548,7 +551,7 @@ Pulp nodes common modules.
 %{python_sitelib}/pulp_node/extensions/__init__.py*
 %{python_sitelib}/pulp_node/*.py*
 %{python_sitelib}/pulp_node_common*.egg-info
-%defattr(640,root,apache,-)
+%defattr(640,apache,pulp,-)
```

Comment:
@dkliban , same here root:pulp

Reply:
Yeah, I think this is true for a lot of places in the spec.